tmk_core/common/eeconfig.c

```diff
@@ -39,7 +39,8 @@ void eeconfig_init_quantum(void) {
   eeprom_update_byte(EECONFIG_DEBUG,          0);
   eeprom_update_byte(EECONFIG_DEFAULT_LAYER,  0);
   default_layer_state = 0;
-  eeprom_update_byte(EECONFIG_KEYMAP,         0);
+  eeprom_update_byte(EECONFIG_KEYMAP_LOWER_BYTE, 0);
+  eeprom_update_byte(EECONFIG_KEYMAP_UPPER_BYTE, 0);
```

Comment:
As mentioned in the latest comments, don't split the bytes. Renamed the existing EECONFIG_KEYMAP memory location to EECONFIG_UNUSED and add a new memory location EECONFIG_KEYMAP at the end that contains the full word.

Reply:
My concern with doing that, is that it will wipe out the settings for everyone that updates.  And ... I don't really like that.  It also pushes the upper limit of the used EEPROM up for no reason other than to make the code simpler. 

The current changes do work, and don't break anything.  It's not not as clean.  

So, my thinking here is "get this in sooner, and then break things later"

Reply:
Alright, I'm ok with that as long as we open a breaking pr soon to fix it